cmd/mev-boost/main.go

```diff
@@ -21,16 +21,18 @@ var (
 	version = "dev" // is set during build process
 
 	// defaults
-	defaultListenAddr         = getEnv("BOOST_LISTEN_ADDR", "localhost:18550")
-	defaultRelayTimeoutMs     = getEnvInt("RELAY_TIMEOUT_MS", 2000) // timeout for all the requests to the relay
-	defaultRelayCheck         = os.Getenv("RELAY_STARTUP_CHECK") != ""
-	defaultGenesisForkVersion = getEnv("GENESIS_FORK_VERSION", "")
+	defaultListenAddr                = getEnv("BOOST_LISTEN_ADDR", "localhost:18550")
+	defaultRelayTimeoutMs            = getEnvInt("RELAY_TIMEOUT_MS", 2000) // timeout for all the requests to the relay
+	defaultRelayCheck                = os.Getenv("RELAY_STARTUP_CHECK") != ""
+	defaultGenesisForkVersion        = getEnv("GENESIS_FORK_VERSION", "")
+	defaultRegisterValidatorInterval = getEnvInt("REGISTER_VALIDATOR_INTERVAL_SEC", 5)
```

Comment:
Let's think about a sane default, if all of the validators send their registration every 5 seconds it will effectively be a DDoS attack on the unfortunate relay on the receiving end
Slot is 12 seconds, that's a sane lower bound
An epoch is 384 seconds, and validators are asked to re-register once per epoch so it's an upper bound
I think something like a fourth (96s) of an epoch is already frequent enough

Reply:
even resending once an epoch should be fine (and instantly on receiving)

Reply:
I agree with your comments, but it seems like it's a bad idea to let the user decide of the interval in the end, wdyt ?